doc/man-sections/client-options.rst

```diff
@@ -68,7 +68,11 @@ configuration.
       auth-user-pass up
 
   If ``up`` is present, it must be a file containing username/password on 2
-  lines. If the password line is missing, OpenVPN will prompt for one.
+  lines. If the password line is missing, OpenVPN will prompt for one on
+  stdin. If you use a graphical OpenVPN client such as OpenVPN GUI on Windows,
```

Comment:
Could we move the "OpenVPN GUI" part as a separate "NOTE" paragraph?  Like this:

    *NOTE (Windows)*:
        If you use a graphical OpenVPN client such as OpenVPN GUI on Windows,
        you should either define both the username and password in the file, or rely
        on the GUI caching the credentials; defining just the username in the file
        will not work.

Reply:
We do similar notes and warning sections other places too, just `git grep` for `NOTE` and `WARNING` to see the formatting.

Reply:
@dsommers done. Looks good now? I did not want to tie this to just Windows as other GUI clients might have the same issue due to same reasons.